### PR TITLE
coq-coqprime : package for coq 8.12

### PR DESCRIPTION
**Pull request by thery**
version for coq 8.12

### CODE REVIEW

File: released/packages/coq-coqprime/coq-coqprime.1.0.6/opam [hunk 1]
```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+maintainer: "thery@sophia.inria.fr"
+homepage: "http://coqprime.gforge.inria.fr/"
+bug-reports: "http://coqprime.gforge.inria.fr/"
```
Comment (palmskog) [inline released/packages/coq-coqprime/coq-coqprime.1.0.6/opam:4]:
```suggestion
homepage: "https://thery.gitlabpages.inria.fr/coqprime/"
bug-reports: "https://github.com/thery/coqprime/issues"
```

Comment (thery) [reply released/packages/coq-coqprime/coq-coqprime.1.0.6/opam]: I prefer not to refer to gitlab

Comment (palmskog) [reply released/packages/coq-coqprime/coq-coqprime.1.0.6/opam]: @thery but this means that the website link will be dead in a few months, as per #1298. What do you want the website for the project to be?

Comment (thery) [reply released/packages/coq-coqprime/coq-coqprime.1.0.6/opam]: The content on the gitlab site is an outdated version of the README.md on gitlhub.

**Pull request merged**